LICENSE

```diff
@@ -199,3 +228,670 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
```

Comment:
@jwilander I copied the same version that Focalboard was using but not sure if this section is required.

Reply:
Yes, having a copy of the AGPL text included here is appropriate.